.buildkite/e2e/nightly-main-matrix.yaml

```diff
@@ -3,9 +3,9 @@
   fixed:
     E2E_PROVIDER: gke
   mixed:
-    - E2E_STACK_VERSION: "7.17.25"
-    # current stack version 8.16.x is tested in all other tests no need to test it again
-    - E2E_STACK_VERSION: "8.18.0-SNAPSHOT"
+    - E2E_STACK_VERSION: "7.17.26"
+    # current stack version 8.17.x is tested in all other tests no need to test it again
+    - E2E_STACK_VERSION: "9.0.0-SNAPSHOT"
```

Comment:
If we advance this to 9.0 we are not testing 8.18 anymore which is not released yet. I wonder if would make sense to test both snapshot versions in the run up to the 9.0 release.